### ISSUE TITLE
attempt to index global 'C_Calendar' (a nil value) on 1.14.2

### ISSUE THREAD

**Issue by trcrsired** [Questie/Questie#5326]
C_Calendar does not exist in 1.14.2. please add some workaround
```
Message: ...AddOns\Questie\Database\Corrections\QuestieEvent.lua:147: attempt to index global 'C_Calendar' (a nil value)
Time: Tue Dec 19 23:05:19 2023
Count: 1
Stack: ...AddOns\Questie\Database\Corrections\QuestieEvent.lua:147: attempt to index global 'C_Calendar' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\Questie\Database\Corrections\QuestieEvent.lua"]:147: in function <...AddOns\Questie\Database\Corrections\QuestieEvent.lua:146>
[string "@Interface\AddOns\Questie\Database\Corrections\QuestieEvent.lua"]:199: in function <...AddOns\Questie\Database\Corrections\QuestieEvent.lua:196>
[string "@Interface\AddOns\Questie\Database\Corrections\QuestieEvent.lua"]:137: in function `Load'
[string "@Interface\AddOns\Questie\Database\Corrections\QuestieCorrections.lua"]:216: in function <...\Questie\Database\Corrections\QuestieCorrections.lua:212>

Locals: 
```
![image](https://github.com/Questie/Questie/assets/100043421/4ba6dadd-539b-4432-a872-91d5f34e8711)

**Comment by trcrsired** [comment]
Here is another one
```
Message: ...face\AddOns\Questie\Modules\Tracker\TrackerUtils.lua:318: attempt to call global 'GetQuestLogCompletionText' (a nil value)
Time: Tue Dec 19 23:16:00 2023
Count: 1
Stack: ...face\AddOns\Questie\Modules\Tracker\TrackerUtils.lua:318: attempt to call global 'GetQuestLogCompletionText' (a nil value)
[string "=[C]"]: in function `GetQuestLogCompletionText'
[string "@Interface\AddOns\Questie\Modules\Tracker\TrackerUtils.lua"]:318: in function `GetCompletionText'
[string "@Interface\AddOns\Questie\Modules\Tracker\QuestieTracker.lua"]:725: in function `_UpdateQuests'
[string "@Interface\AddOns\Questie\Modules\Tracker\QuestieTracker.lua"]:1632: in function `Update'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestEventHandler.lua"]:480: in function `QuestRelatedFrameClosed'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestEventHandler.lua"]:576: in function <...e\AddOns\Questie\Modules\Quest\QuestEventHandler.lua:541>

Locals: 
```
![image](https://github.com/Questie/Questie/assets/100043421/3cd44412-6409-40bf-8917-ac043c248bbb)

**Comment by MyTechnoHunter** [comment]
All official game clients have the API in question.. you will need to look elsewhere for the answer or revert to an older release of questie

**Comment by BreakBB** [comment]
Hey @trcrsired as TechnoHunter already said: We only support the official Blizzard game clients. With the size of the team, we are already at the edge of client numbers we can support and simply can not watch out for all kind of private server versions, sorry.

But as also said: Feel free to revert to an older (for you) working version of Questie.

**Comment by trcrsired** [comment]
Hello. Can i fix myself and PR? I can fix it for you

**Comment by BreakBB** [comment]
Hey @trcrsired sure thing!

**Comment by BreakBB** [comment]
Fixed by #5340
